routes/api.php

```diff
@@ -73,6 +74,15 @@ function () {
                 Route::put('notifications/read/{id}', 'NotificationController@readNotification')
                     ->where('id', '[0-9]+');
 
+                // Prenotes
+                Route::prefix('prenote-messages')->group(function () {
```

Comment:
If this was entirely RESTful, shouldn't this be `prenotes/messages`?

Reply:
Good shout, 8dcf7cfc77abc6ae89cf0647bd75e91715eeda27